README.md

```diff
@@ -49,3 +49,20 @@ Similarly, each dependency denotes the version of that dependency used and a lin
 #### How to Onboard a New Repository
 
 If you would like to see your repository on BARViz, it needs to be published to the Build Asset Registry. Please see the [Darc documentation](https://github.com/dotnet/arcade/blob/master/Documentation/Darc.md).
+
+## Validation Process in dev and int environments
+
+For any non-deployment code changes, the expectation is to have run the tests corresponding to the service locally to confirm that the change works before putting up the PR. The tests for each of the major areas in arcade-services are as below:
+- [Maestro](src\Maestro\Tests)
+- [Darc](src\Microsoft.DotNet.Darc\tests)
+- [Scenario Tests](src\Maestro\tests\Scenarios)
+
+For any deployment changes, the only way to test would be to kick off the [build pipeline](https://dev.azure.com/dnceng/internal/_build?definitionId=252&_a=summary) that deploys the intended service from the dev branch to staging / int environment.
+
+>This comes with a significant overhead of a possibility of leaving the int deployments in a broken or hung state, which then would require significant manual effort to undo the damage especially with the Service Fabric Clusters. This process should only be done if and only if absolutely necessary and after obtaining management approval.
```

Comment:
This renders as a quote, would it be better to render it in bold instead?

Reply:
I made the statements in bold as well, I have left them in a quote coz it puts it in a nice little box.

Reply:
Changed it a table with a heading of Warning in all bold, looks a lot better now :)